crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -41,17 +30,16 @@ async fn main() -> Result<()> {
 }
 
 async fn run() -> Result<()> {
-    let args = Cli::parse();
+    let conf = match config::get() {
+        Ok(cfg) => cfg,
+        Err(e) => {
+            eprintln!("failed to read configuration: {e}");
+            std::process::exit(2);
+        }
+    };
+
     // hierarchical config. cli args override Envars which override toml config values
```

Comment:
This comment is no longer valid if we get rid of cli args. It could be edited and moved up though.